build_debian.sh

```diff
@@ -205,6 +205,8 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install      \
 ## Note: don't install python-apt by pip, older than Debian repo one
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install      \
     file                    \
+    ifupdown2               \
```

Comment:
ifupdown is installed with a newer version via #3180

Reply:
As long as ifupdown2 is removed from here per Shuotian's comment, the rest looks good to me.

Reply:
@nikos-github ifupdown2 is removed